metrics/grafana/pd.json

```diff
@@ -2340,6 +2340,113 @@
             "alignLevel": null
           }
         },
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "description": "The allowance status of the scheduling.",
```

Comment:
How about putting is near "Scheduler is running"
But it makes sense where it is now.

Reply:
I prefer leaving it here since it's more like a cluster-level status rather than the scheduler. Another reason is that if it is placed in the `Scheduler` panel, it may cause many changes to the Grafana JSON file. If it is only appended here, there will be fewer changes.